specification/recoveryservicesbackup/resource-manager/Microsoft.RecoveryServices/stable/2016-12-01/bms.json

```diff
@@ -4110,12 +4110,7 @@
             "VMAppContainer",
             "SQLAGWorkLoadContainer",
             "StorageContainer",
-            "GenericContainer",
```

Comment:
Why are these being removed from a stable spec?  This is a breaking change from an SDK perspective.  Same comment applies to other places where enums are removed.

Reply:
When a swagger is merged into the public repo it's considered a release event.  As such we've already released this for the Go programming language and possibly others.  While we won't prevent you from making this change it will be flagged as a breaking change and reported in S360.

Reply:
Do you want to leave these breaking changes as-is?

Reply:
Yes. We discussed this internally. These were not used anywhere. So we will leave them as is.